doc/UsersGuide/source/Backgroundinfo/TechnicalOverview.rst

```diff
@@ -91,3 +91,260 @@ HAFS Workflow Schematic
 .. figure:: https://github.com/hafs-community/HAFS/wiki/docs_images/hafs_workflow_schematic.png
     :width: 75 %
     :alt: HAFS Workflow Schematic
+
+.. _OperationalUpdates:
+
+===============================
+HAFS v2.0.0 Operational Updates 
+===============================
+
+**Model:** Hurricane Analysis and Forecast System (HAFS)
+
+**Version:** hafs.v2.0.0
+
+**Released:** July 01, 2024, 1200UTC
+
+**Location on GitHub:** `HAFS GitHub Repository <https://github.com/hafs-community/HAFS/tree/production/hafs.v2>`
+
+**Clone the repository:**
+
+.. code-block:: bash
+
+   git clone -b production/hafs.v2 --recursive  https://github.com/hafs-community/HAFS.git ./hafs.v2.0.0
+
+**Purpose:**
+    Run :term:`HAFS`.v2 :term:`HFSA` configuration to provide hurricane track and intensity forecast guidance to 5.25 days over all global oceanic basins, including :term:`NATL`, :term:`EPAC`, :term:`CPAC`, :term:`WPAC`, :term:`NIO`, and :term:`SH` basins. Run :term:`HAFS`.v2 :term:`HFSB` configuration to provide hurricane track and intensity forecast guidance for 5.25 days over :term:`NHC`/:term:`CPHC` basins, including :term:`NATL`, :term:`EPAC`, :term:`CPAC` basins.
+
+
+**Developed by:**
+    EMC and the UFS Hurricane Application Team. See currently active HAFS developers below.
+
+.. list-table:: HAFS Developers
```

Comment:
@jdkublnick Now that I'm looking at the human-readable table on RTD, it might be nice to merge repeated cells. Let's discuss!